tsdb/encoding/encoding.go

```diff
@@ -184,23 +191,27 @@ func (d *Decbuf) Skip(l int) {
 }
 
 func (d *Decbuf) UvarintStr() string {
-	return string(d.UvarintBytes())
+	return string(d.UvarintBytes(nil))
 }
 
 // The return value becomes invalid if the byte slice goes away.
```

Comment:
> // The return value becomes invalid if the byte slice goes away.

I think this is not valid, it's obsolete comment, and it was obsolete even before this change, as we always copy, no? :thinking:

Reply:
Now that we are re-using the slice, it _makes sense_ because if slice does not exist, then there is no guarantee about the data :)